app/helpers/application_helper.rb

```diff
@@ -150,9 +150,11 @@ def make_filterable_column(title, options = nil, *columns)
     raise 'Must have at least one column' if columns.empty?
 
     html_classes = ['filterable']
-    content = content_tag(:span, title, :class => :title)
-    selected = @query.blank? || columns.any? { |c| @columns.include?(c) }
-    options ||= {}
+    content      = content_tag(:span, title, :class => :title)
+    options    ||= {}
```

Comment:
Ahora si, "Con Elvira es otra cosa" =P

No pude evitar poner el tema después de nombrarlo... te lo dejo por si te pasa: https://youtu.be/SIFzR1W6hkE =P

Reply:
Le gusta estar al sol? No tiene ni sentido la canción jaja. Quizás tiene otro significado que no estoy entendiendo xD

Reply:
Jajaja, si, muy abstracto. En persona lo aclaro mejor, a texto puro va a ser difícil.